setup.sh

```diff
@@ -9,3 +9,7 @@ cp -r shared 8.5-jre8-alpine/tmp/shared
 rm -rf 9.0-jre8-alpine/tmp/shared
 mkdir -p 9.0-jre8-alpine/tmp
 cp -r shared 9.0-jre8-alpine/tmp/shared
```

Comment:
Please remove the apline folders since those won't be supported

Reply:
Keeping it for a while for reference, if needed.